src/util/util_errors.c

```diff
@@ -115,6 +115,7 @@ struct err_string error_to_str[] = {
     { "Unable to initialize SSL" }, /* ERR_SSL_FAILURE */
     { "Unable to verify peer" }, /* ERR_UNABLE_TO_VERIFY_PEER */
     { "Unable to resolve host" }, /* ERR_UNABLE_TO_RESOLVE_HOST */
+    { "UID outside range" }, /* ERR_UID_OUTSIDE_RANGE */
```

Comment:
Here is a nitpick, I'd like to know whether "UID is outside of allowed range" would be more clear.

Reply:
We should also not use UID but "ID" because we use the error code also for group GIDs

Reply:
@fidencio @jhrozek Done
{ "ID is outside of allowed range" }, /* ERR_ID_OUTSIDE_RANGE */